### DISCUSSION TITLE
DNS-over-HTTPS change port

### DISCUSSION THREAD

**Discussion by eyeblve** [AdguardTeam/AdGuardHome#5379] [Q&A]
iOS and macOS configuration

DNS-over-HTTPS does not work when I change port 443 to another

Is it possible to change port 443?

**Comment by ammnt** [comment]
> iOS and macOS configuration
> 
> DNS-over-HTTPS does not work when I change port 443 to another
> 
> Is it possible to change port 443?

Yes, you can just edit you .mobileconfig file via text editor like Notepad++, SublimeText etc. and change the 'ServerURL' row:

```
<key>ServerURL</key>
<string>https://example.com:443/dns-query</string>
```

**Reply by ghost** [reply]
> > iOS and macOS configuration
> 
> > 
> 
> > DNS-over-HTTPS does not work when I change port 443 to another
> 
> > 
> 
> > Is it possible to change port 443?
> 
> 
> 
> Yes, you can just edit you .mobileconfig file via text editor like Notepad++, SublimeText etc. and change the 'ServerURL' row:
> 
> 
> 
> ```
> 
> <key>ServerURL</key>
> 
> <string>https://example.com:443/dns-query</string>
> 
> ```

Nice tip!

**Comment by eyeblve** [comment]
It does not work, with different configurations, changing the port in the encryption section and neither, when I close port 443 it does not work
